paper.tex

```diff
@@ -192,27 +192,49 @@ \section*{Introduction}
 
 \section*{Demes}
 
-\begin{itemize}
-    \item Demographic models are comprised of one or more interacting
-        populations, along with description, doi, time units and generation
-        times.
-    \item Populations (or demes) are treated as discrete collection of
-        exchangeable individuals following a well-defined set of rules
-    \item These rules include population sizes, times of existence, and other
-        information about the mating system (e.g. selfing or cloning rates)
-    \item Relationships between demes are defined by ancestors/descendents
-        and possible continuous or instantaneous migration between coexisting
-        demes
-\end{itemize}
-
-\subsection*{High-level model specification in YAML}
-
-\begin{itemize}
-    \item YAML allows us to write this information in a human-readable and
-        structured manner (lists, keys/items, etc)
-    \item Implicit values, so that models can be written compactly
-    \item Example of an IM Model~\ref{code:im_model}
-\end{itemize}
+The Demes specification is a formal data model for describing
+the properties of populations over time,
+along with some metadata and provenance information.
+The data model is based on the ubiquitous JSON~\citep{bray2017javascript}
+standard, and the structure formally defined using
+JSON Schema~\citep{wright2020json}.
```

Comment:
This isn't strictly true (at least, not right now). In particular, we use infinity as a numeric value, which JSON doesn't allow (and by extension, a strictly correct JSON Schema validator shouldn't work with inifinite values). Are we planning to update the spec to use a union of numbers and the string "Infinity" for "start_time" params? I guess we should to that, but it's a bit messy. The alternative is to say in the spec that we do the string "Infinity" thing only when JSON is used as the serialisation format (which is less formal, but also less work).

Reply:
I think we should try to be formally correct with JSON Schema - there's no much point in having a schema if we can't use it to validate the documents. Can you open an issue on the spec repo to track, please?

Reply:
I think we should fix this: https://github.com/popsim-consortium/demes-spec/issues/79

Let's take the detailed discussing over to the spec repo.